index.org

```diff
@@ -103,51 +103,58 @@ novice]] and [[http://howardism.org/Technical/Emacs/literate-devops.html][powerf
 
 * Further reading
 
-As well at the [[file:quickstart.org][quickstart]] and [[file:manual/index.html][manual]], there's also
-a [[file:guide/index.html][guide]], and a wealth of community-written docs on [[https://orgmode.org/worg/][Worg]] .
+Beyond the [[file:quickstart.org][quickstart]] and the [[file:manual/index.html][manual]],
+Org has a [[file:guide/index.html][guide]], and a wealth of community-written documentation on [[https://orgmode.org/worg/][Worg]] .
 
-Once you've used Org for a bit, you'll know it's great --- it's actually
-so great that there have been [[https://orgmode.org/worg/org-papers.html][(scientific) papers]] written about the value of Org
-for [[https://en.wikipedia.org/wiki/Reproducibility#Reproducible_research][reproducible research]].
+Nearly every Org user has a story to tell about how Org enables and empowers them ---
```

Comment:
Hmm. Is this an opportunity to turn "story" "to" and "tell" into links to such stories? Just an idea.

Reply:
Yes. This was one of the places where I really wanted to be able to link to a gallery or other user stories.

Reply:
Cool. I haven't got any links on hand, perhaps this is something to come back to later?

Reply:
I think we are just going to leave the gallery out for now.